native/src/utils/NativeLanguageDetector.ts

```diff
@@ -1,17 +1,28 @@
 import { getLocales } from 'react-native-localize'
+import { config } from 'translations'
 
 export default {
```

Comment:
Apparently you could also type this as `LanguageDetectorModule` now. Then you can also see that you can omit `async: false`, as there is a separate `LanguageDetectorAsyncModule type. But also feel free to keep it as it is :)